### PR TITLE
Created landing page

### LINKED ISSUE #21: create landing page

### PR DESCRIPTION
**Pull request by msclal**
image

closes issue #21 
@minsooerickim 

- copied svg code from figma to get exact asset layout for landing
- still need to add color and font
- landing page text is in countdown component instead of landing.tsx (dont know if code-wise thats the best practice)

### CODE REVIEW

File: components/SignupCounter.tsx [hunk 1]
```diff
@@ -25,11 +25,11 @@ export default function SignupCounter() {
   if (error || !data) {
     return (
       <div className="flex items-center gap-2">
-        <div>
+        {/* <div>
           <BiEdit className="text-2xl xs:text-3xl" />
```
Comment (minsooerickim) [inline components/SignupCounter.tsx]: you can just get rid the old code here; no need to comment

o also, if u remove the `import { BiEdit } from 'react-icons/bi';` at the top, it'll fix the vercel deployment. ( it's complaining about unused variables)

File: components/SignupCounter.tsx [hunk 2]
```diff
@@ -40,25 +40,25 @@ export default function SignupCounter() {
...
-          &nbsp;hackers signed up so far!
+          &nbsp;hackers signed up!
         </h4>
       </div>
     );
   } else {
     return (
       <div className="flex items-center gap-2">
-        <div>
+        {/* <div>
```
Comment (minsooerickim) [inline components/SignupCounter.tsx]: same thing here, i think u can jus remove the old code here

File: pages/sections/landing.tsx [hunk 3]
```diff
@@ -61,7 +70,7 @@ export default function Landing() {
                 </motion.button>
               </Link>
             )}
-          {!session && (
+          {/* {!session && (
```
Comment (minsooerickim) [inline pages/sections/landing.tsx:74]: vercel is also complaining about the `toggleSigninModal` function on line 21.

I think u can just comment out the `toggleSigninModal` function (lines 21-23) to make the vercel deployment pass

**Review by minsooerickim — CHANGES_REQUESTED** [review]
landing looks sooo goood 💯

One more comment, instead of creating a component (`landing_asset.tsx`) can you save the SVG file as a .svg file and import it into landing.tsx with a image tag (preferably next/image's Image tag)

**Comment by minsooerickim** [comment]
For the text in countdown, ideally, like u said, the text would go in `index.tsx ` and the countdown component would just have the countdown. This stresses the idea of a "component based framework". This way, if we have other places that we need to use the countdown for, we can simply import the countdown as a COMPONENT and use it without having to modify the text or anything.

BUTTT for our case, since we just use the countdown in one place, i think its fine to leave it as is. but its up to u :)))

**Comment by msclal** [comment]
image
@minsooerickim 

- colors on buttons, navbar, countdown looks naked and jank
- tried changing colors but realized its not a quick ez-pz lemon squeezy. from what i've gathered, light and dark mode themes are toggled  with ```next-theme``` and ```global.css```. specifically, ```[data-theme='light']``` and ```[data-theme='dark']```  in ```global.css``` are what distinguishes dark/light mode colors. but with mariam's color pallet implementation, global.css is not being used and instead we're using ```tailwind.config.js``` with no separation between light and dark themes. so i think, we need to specify the light and dark mode colors in ```tailwind.config.js``` and make it work with ```next-theme``` 
- also marketing mightve missed some hex colors in the figma. the landing page font colors is not listed
- overall landing page is goooddd it just needs the colors and font. we might need to open another issue to fix the color pallete stuff and offload them to another person so i can get the schedule page going😅  (although if need be i can also look into it)

**Comment by minsooerickim** [comment]
> * mightve missed some hex colors in the fig

yea mariam's hex color changes in `tailwind.config.js` need to be mapped to a variable and the hex codes shud b in `globals.css`. but i just merged it in since i didn't think it was a big deal for now. 

I don't think u have to worry about colors too much rn. I think UIUX and marketing are still messing around with colors in some parts and colors r p ez to fix later on as well so I think ur guudddd.

**Comment by minsooerickim** [comment]
it seems like the asset doesn't fill the screen on my monitor (1920 x 1080); I remember it looked fine when u had the SVG as a component tho 🤷‍♂️

i think other screen sizes r fine tho. generally, i check for mobile, iPad, laptop, and monitor screen sizes (it seems like ur changes work fine with everything but the monitor size)
image

**Comment by minsooerickim** [comment]
> it seems like the asset doesn't fill the screen on my monitor (1920 x 1080); I remember it looked fine when u had the SVG as a component tho 🤷‍♂️
> 
> i think other screen sizes r fine tho. generally, i check for mobile, iPad, laptop, and monitor screen sizes (it seems like ur changes work fine with everything but the monitor size) image

ik this can get hella annoying so if it dis is too hard to fix, u can honestly jus revert back to the component method as u did before (i vaguely remember it working on monitor res)

**Comment by msclal** [comment]
> > it seems like the asset doesn't fill the screen on my monitor (1920 x 1080); I remember it looked fine when u had the SVG as a component tho 🤷‍♂️
> > i think other screen sizes r fine tho. generally, i check for mobile, iPad, laptop, and monitor screen sizes (it seems like ur changes work fine with everything but the monitor size) image
> 
> ik this can get hella annoying so if it dis is too hard to fix, u can honestly jus revert back to the component method as u did before (i vaguely remember it working on monitor res)

mmmmm interesting. ill try to make the .svg work if not ill change it back to having it as a component 🫡

**Comment by msclal** [comment]
image

attempt #90139190238 🤞🏼

**Comment by minsooerickim** [comment]
@msclal ur crackedd 😱😱😱😱😱😱😱😱😱

**Pull request merged**